cg/services/file_delivery/deliver_files_service/deliver_files_service.py

```diff
@@ -73,3 +83,32 @@ def deliver_files_for_ticket(
             self.deliver_files_for_case(
                 case=case, delivery_base_path=delivery_base_path, dry_run=dry_run
             )
+
+    def _start_rsync_job(self, case: Case, dry_run: bool, folders_to_deliver: set[Path]) -> int:
+        job_id = self.rsync_service.run_rsync_for_case(
+            case=case,
+            dry_run=dry_run,
+            folders_to_deliver=folders_to_deliver,
+        )
+        self.rsync_service.write_trailblazer_config(
+            content={"jobs": [str(job_id)]},
+            config_path=self.rsync_service.trailblazer_config_path,
+            dry_run=dry_run,
+        )
+        return job_id
+
+    def _add_trailblazer_tracking(self, case: Case, job_id: int, dry_run: bool) -> None:
+        if not dry_run:
+            analysis: TrailblazerAnalysis = self.tb_service.add_pending_analysis(
+                case_id=case.internal_id,
+                analysis_type=AnalysisTypes.OTHER,
+                config_path=self.rsync_service.trailblazer_config_path.as_posix(),
+                order_id=case.latest_order.id,
+                out_dir=self.rsync_service.log_dir.as_posix(),
+                slurm_quality_of_service=Priority.priority_to_slurm_qos().get(case.priority),
+                workflow=Workflow.RSYNC,
+                ticket=case.latest_ticket,
+            )
+            self.tb_service.add_upload_job_to_analysis(analysis_id=analysis.id, slurm_id=job_id)
+            self.analysis_service.add_upload_job(slurm_id=job_id, case_id=case.internal_id)
+        LOG.info(f"Transfer of case {case.internal_id} started with SLURM job id {job_id}")
```

Comment:
This logging  seems a bit out of place? Or this is not where we start the transfer right?